### DISCUSSION TITLE
how store Array

### DISCUSSION THREAD

**Discussion by Dalal5** [livewire/livewire#3040] [7. Bugs (archived)]
how store Array  using livewire?

public $data=[
[
'section_type'=>'' ",
'type'   =>'' ",

]

];

**Comment by calebporzio** [comment]
Please provide a more clear question and code sample. There's not enough here to help

**Comment by kondratovbr** [comment]
Not sure what you meant, but I stumbled upon this issue and just want to warn you right ahead: I believe `$data` is a reserved property in Livewire components, used internally, so you shouldn't use it for your own data and you also shouldn't tamper with it, or you may encounter some nasty heisenbugs. Though I may be totally wrong, but worth a try if your stuck with some weird issue.
